e2e/cypress/integration/enterprise/system_console/support_packet_generation_spec.js

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @enterprise @system_console
+
+// # Goes to the System Scheme page as System Admin
+const goToAdminConsole = () => {
+    cy.apiAdminLogin();
+    cy.visit('/admin_console');
+};
+
+describe('Support Packet Generation', () => {
+    it('MM-T3817-Step1 - Commercial Support Dialog UI', () => {
+        // # Remove license
+        cy.apiAdminLogin();
+        cy.apiDeleteLicense();
+
+        // # Go to System Console
+        goToAdminConsole();
+
+        cy.findByRole('button', {name: 'Menu Icon'}).should('exist').click();
+
+        // * Make sure the commercial support button takes you to external link
+        cy.findByRole('link', {name: 'Commercial Support'}).and('have.attr', 'href').and('include', '/commercial-support/');
+    });
+
+    it('MM-T3817-Step2And3 - Commercial Support Dialog UI', () => {
```

Comment:
I merged step 2 and step 3 into one step and created new ticket MM-T3849 to match cypress case

```suggestion
    it('MM-T3849 - Commercial Support Dialog UI - E10/E20 License', () => {
```